### DISCUSSION TITLE
dataset数据集问题

### DISCUSSION THREAD

**Discussion by ChenXuuuuu** [open-mmlab/mmdetection#8825] [Q&A]
我的数据集train中由882张图像，但是训练时控制台打印出了[**/5292]，跟882是一个六倍的关系，这正常吗，为什么呢？是哪里有什么参数吗？

**Comment by jbwang1997** [comment]
If you use the `RepeatDataset` to enlarge the dataset?

**Comment by vansin** [comment]
please provide the config file ~
